GPU/GPUTracking/DataTypes/GPUDataTypes.h

```diff
@@ -211,7 +212,10 @@ struct ConstPtr {
 
 template <template <typename T> class S>
 struct GPUCalibObjectsTemplate {
+  float transformScale = 0;
```

Comment:
The transformScale must not be in here, this struct must contain only pointers to POD or to FlatObjects. Could you move it to CorrectionMapsHelper? And perhaps leave a comment in this struct to only have pointers to either POD or FlatObjects in here?
(The correctionMapsHelper will be a special case. It is not a POD but has to pointers, but these point to 2 flatobjects inside the struct, so will be fine with some manual handling.)

Reply:
thanks for spotting, this is actually a leftover from an older version, not used.